docs/tutorial-3.md

```diff
@@ -0,0 +1,162 @@
+# How to include external Actions
+
+Now an [example rust project](https://github.com/input-output-hk/cicero-example/tree/main/rust) is used, to explain how to setup an external repository with Cicero.
+
+We will call this an external Action, because the Action will be provided by a repository which isn't the Cicero repository itself.
+
```

Comment:
I do not see information about https://github.com/input-output-hk/cicero-example/blob/main/rust/cicero/lib.nix file. I think it is cicero standard library. It would be good to say that it is neccessary. I assume it is right?

Reply:
https://github.com/input-output-hk/cicero/blob/main/lib.nix is the actual standard library from Cicero.

https://github.com/input-output-hk/cicero-example/blob/main/rust/cicero/lib.nix just extends the library to write the Action in a more concise way.